recipe/meta.yaml

```diff
@@ -20,16 +20,16 @@ requirements:
     - python >=3.7
     - pip
     - pyct >=0.4.4
-    - param >=1.7.0,<2.0
+    - param >=1.12.0,<3.0
```

Comment:
Interesting to see that this pinned like this `- param >=1.7.0,<2.0` before, while in HoloViews it's `"param >=1.7.0"`. To build GeoViews we don't really need to pin Param as this is just to get the version.

@Hoxbro up to you to see whether you want to change that, that's pure nitpick.

Reply:
Sorry indeed I meant HoloViews.

Yep having project.toml, setup.py (which is now used by conda-build to get the build extras and populate the recipe) and the conda recipe should in a dream world be aligned. It makes it more likely for the conda packages that ends up on pyviz and conda-forge to be identical.